spec/requests/registrations_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Registration', type: :request do
+  describe 'Email registration' do
+    let(:sign_up_url) { '/auth' }
+    let(:password) { Faker::Internet.password }
+    let(:sign_up_params) do
+      {
+        email: Faker::Internet.email,
+        password: password,
+        password_confirmation: password,
+        gender: Faker::Gender.type
+      }
```

Comment:
I recommend using user factory here and `attributes_for` instead. Not a blocker. https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#build-strategies